uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/KeyCloakGroupsManager.java

```diff
@@ -0,0 +1,155 @@
+            RoleRepresentation roleRepresentation = getRoleRepresentation(identifier, roleResource);
+            return createGroup(roleRepresentation);
+        }
+        throw new GroupNotFoundException(identifier);
+    }
+    
+    @Override
+    public Group create(Group entity) throws GenericUsersSystemManagementException {
+        if (entity == null) throw new NullPointerException();
+        RealmResource realmResource = getRealmResource();
+        RolesResource rolesResource = realmResource.roles();
+        RoleRepresentation roleRepresentation = new RoleRepresentation();
+        roleRepresentation.setName(entity.getName());
+        rolesResource.create(roleRepresentation);
+        return entity;
+    }
+
+    // The Group class from Errai does not hava any attributes holder. Group name cannot be modified for a group. 
+    // So currently a group cannot be updated.
+    @Override
+    public Group update(Group entity) throws GenericUsersSystemManagementException {
+        throw new UnsupportedServiceCapabilityException(Capability.CAN_UPDATE_GROUP);
+    }
+
+    @Override
+    public void delete(String... identifiers) throws GenericUsersSystemManagementException {
+        if (identifiers == null) throw new NullPointerException();
+        RealmResource realmResource = getRealmResource();
+        RolesResource rolesResource = realmResource.roles();
+        for (String identifier : identifiers) {
+            RoleResource roleResource = rolesResource.get(identifier);
+            if (roleResource == null) throw new GroupNotFoundException(identifier);
+            roleResource.remove();
+        }
+    }
+
+    @Override
+    public void assignUsers(String name, Collection<String> users) throws GenericUsersSystemManagementException {
+        if (name == null) throw new NullPointerException();
+        if (users != null) {
+            RealmResource realmResource = getRealmResource();
+            UsersResource usersResource = realmResource.users();
+            RolesResource rolesResource = realmResource.roles();
+            RoleResource roleResource = rolesResource.get(name);
+            List<RoleRepresentation> rolesToAdd = new ArrayList<RoleRepresentation>(1);
+            rolesToAdd.add(getRoleRepresentation(name, roleResource));
+            for (String username : users) {
+                UserResource userResource = getUserResource(usersResource, username);
+                if (userResource == null) throw new UserNotFoundException(username);
+                userResource.roles().realmLevel().add(rolesToAdd);
+            }
+        }
+
+    }
+
+    @Override
+    public CapabilityStatus getCapabilityStatus(final Capability capability) {
+        if (capability != null) {
+            switch (capability) {
+                case CAN_SEARCH_GROUPS:
+                case CAN_ADD_GROUP:
+                case CAN_READ_GROUP:
+                case CAN_DELETE_GROUP:
+                    return CapabilityStatus.ENABLED;
```

Comment:
I think adding a default case here is cleaner and gets rid of the warning. case default: return CapabilityStatus.UNSUPPORTED or simply break.